app/Models/Presenters/CompanyPresenter.php

```diff
@@ -36,7 +36,11 @@ public function logo($settings = null)
             $settings = $this->entity->settings;
         }
 
-        return (strlen($settings->company_logo) > 0) ? url('') . $settings->company_logo : 'https://www.invoiceninja.com/wp-content/uploads/2019/01/InvoiceNinja-Logo-Round-300x300.png';
+        if(strlen($settings->company_logo))
+            return asset($settings->company_logo);
```

Comment:
Hi, since update to this version v5.1.12, the logo is not displayed anymore on invoices.
It's OK with : 
`return url('') . $settings->company_logo;`